libr/core/cmd_debug.c

```diff
@@ -792,8 +792,14 @@ static void cmd_debug_modules(RCore *core, int mode) { // "dmm"
 				//r_cons_printf ("fs+module_%s\n", fn);
 				r_cons_printf ("f mod.%s = 0x%08"PFMT64x"\n",
 						fn, map->addr);
+#if __WINDOWS__
```

Comment:
we should escape the map->file path here, i know this is out of the scope of the PR, just something TODO (if you mind in this PR)